app/models/package_manager/pypi/json_api_project.rb

```diff
@@ -68,7 +68,10 @@ def repository_url
 
       def homepage_url
         @data.dig("info", "home_page").presence ||
```

Comment:
🎱 Is it worth logging if we can't find a homepage so we can go back and see if there are additional variations of "Homepage" we can pull from?